guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,38 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
+
+{Team} recommends performing operating system updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
+
+If there are any released {ProjectServer} updates, those will be included when performing this update.
+We do not recommend updating the operating system independently of {ProjectServer}.
+Disconnected users should ensure that their installation is pointing at up to date {ProjectServer} repositories.
+
+.Prerequisites
+* Back up your {ProjectServer}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+
+.Procedure
+. Use the health check option to determine if the system is ready for update.
+On first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} update check
+----
```

Comment:
Why are we mentioning health check but showing update check?

Reply:
Because `update check` is essentially a health check. Just not *the* health check `f-m health check`.

And probably because it's copied from the "updating project" chapter.